Sources/CNIOSHA1/c_nio_sha1.c

```diff
@@ -235,10 +233,7 @@ c_nio_sha1_pad(ctxt)
 }
 
 void
-c_nio_sha1_loop(ctxt, input, len)
-	struct sha1_ctxt *ctxt;
-	const uint8_t *input;
-	size_t len;
+c_nio_sha1_loop(struct sha1_ctxt *ctxt, const uint8_t *input, size_t len)
```

Comment:
OMG, they migrated from K&R C to ANSII C!!! They caught up with 1989!!! the future